css/styles.css

```diff
@@ -534,6 +538,75 @@ Footer
   margin-bottom: 4rem;
 }
 
+/*
+=====================================
+Popup
+=====================================
+*/
+
+.popup {
+  background-color: #1617174d;
+  -webkit-backdrop-filter: blur(10px);
+  backdrop-filter: blur(10px);
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 200;
+  width: 100%;
+  height: 100vh;
+  padding: var(--horizontal-spacing);
+  display: none;
+}
+
+.popup__bg {
+  background-color: white;
+  padding: var(--horizontal-spacing);
+  border-radius: 1rem;
+  height: 100%;
+  overflow-y: scroll;
+  max-width: 900px;
+  margin-inline: auto;
+}
```

Comment:
Kindly ensure that the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). At certain screen widths of the mobile version, there are some slight variations between your design and the figma in terms of the position and margins of the `see live` and `see source` buttons as shown below :point_down: 

### Your design 

![Screenshot from 2022-12-20 20-48-31](https://user-images.githubusercontent.com/65233860/208764665-1a49408b-d5ff-456a-8324-d0f09f1d4fb1.png)


### Figma design

![Screenshot from 2022-12-20 20-47-54](https://user-images.githubusercontent.com/65233860/208764706-cfc68b94-8c84-4484-b965-533556dd8f60.png)